### DISCUSSION TITLE
SSH connection

### DISCUSSION THREAD

**Discussion by exoton** [magiblot/tvision#76] [Q&A] [answered]
Hello. Thank you for your hard work. I've used it in the early 90's, and I loved it.
Will your port work over SSH connection?

**Comment by magiblot** [comment, marked as answer]
Hi exoton, thanks for your comment. It should be already working over SSH, at least on Unix systems.

**Reply by WillianBR** [reply]
@magiblot  and @exoton My drafts are working nicely under SSH connection. I can use mouse and keyboard without any problem.
